src/main/scala/com/typesafe/sbt/packager/jdkpackager/JDKPackagerHelper.scala

```diff
@@ -62,40 +42,79 @@ object JDKPackagerHelper {
     basename: String,
     iconPath: Option[File],
     outputDir: File,
-    sourceDir: File) = {
+    sourceDir: File): Map[String, String] = {
 
-    def iconArg = iconPath
+    val iconArg = iconPath.toSeq
       .map(_.getAbsolutePath)
-      .map(p ⇒ Map(s"-Bicon=$p" -> ""))
-      .getOrElse(Map.empty)
+      .map(p ⇒ s"-Bicon=$p")
 
-    def mainClassArg = mainClass
+    val mainClassArg = mainClass
       .map(c ⇒ Map("-appclass" -> c))
       .getOrElse(Map.empty)
 
-    //    val cpSep = sys.props("path.separator")
-    //    val cp = classpath.map(p ⇒ "lib/" + p)
-    //    val cpStr = cp.mkString(cpSep)
+    // Make a setting?
+    val jvmOptsFile = (sourceDir ** "jvmopts").getPaths.headOption.map(file)
```

Comment:
My vote would be for a setting that specifies the JVM options as a `Seq[String]`, and not rely on some external source file with some one-off format (fine with me if it's converted to a source file as a part of `universal:packageBin`.) Almost went that route with this plugin, but I worked hard to avoid introducing any new constructs here.